### DISCUSSION TITLE
Ender 3 Pro conversion to use Jyers

### DISCUSSION THREAD

**Discussion by codeframe78** [Jyers/Marlin#1390] [Q&A] [answered]
Hey guys,

I am going to be upgrading my 8 bit E3P to a 32 bit 4.2.7 silent board. I am currently running Jyers on both of my E3v2s and love it, and I would like to use it with the upgraded E3P when I swap out the board.  I also purchased a display for the E3v2. My question is, with the board and screen update, would I be able to run Jyers on this machine?

**Comment by blackoutlaw** [comment, marked as answer]
yes you can i have a e3p and did the upgrade and it works great acts and looks like a ev2
